src/cephadm/cephadm

```diff
@@ -2014,10 +2014,12 @@ def command_deploy():
         monitoring_args = [
             '--user',
             str(uid),
-            '--cpus',
-            metadata.get('cpus', '2'),
-            '--memory',
-            metadata.get('memory', '4GB')
+            # FIXME: disable cpu/memory limits for the time being (not supported
+            # by ubuntu 18.04 kernel!)
+            #'--cpus',
+            #metadata.get('cpus', '2'),
+            #'--memory',
+            #metadata.get('memory', '4GB')
```

Comment:
I wonder if we should be passing the cpus/memory args via `--config-json` and use them only if the key exists?

Reply:
Yeah, agree